Makefile.validation

```diff
@@ -49,6 +49,17 @@ test-sanity:
 #	should fail when requesting to create a volume with already existing name and different capacity|\
 #	should not fail when requesting to create a volume with already existing name and same capacity" -ginkgo.v -timeout=0
 
+load-helm-charts:
```

Comment:
This target must be in private repository.

Reply:
removed